cmd/status.go

```diff
@@ -73,7 +73,7 @@ func Status(fs afero.Fs) *cobra.Command {
 				return err
 			}
 
-			manifestOpts := contextCMD.ManifestOptions{Filename: devPath, Namespace: namespace, K8sContext: k8sContext}
+			manifestOpts := contextCMD.ManifestOptions{Filename: devPath}
```

Comment:
If we don't use it anymore. Should we remove it completely from the `ManifestOptions`?

Reply:
As talked offline, removed from ManifestOptions Namespace and K8sContext as they are not used